docs/guides/create-account-non-partner.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
@AngeloCG97  we have removed this file completely

rebase your branch with master

Reply:
create-account-user